src/data/polynomial/hasse_deriv.lean

```diff
@@ -120,9 +120,9 @@ lemma factorial_smul_hasse_deriv :
   ⇑(k! • @hasse_deriv R _ k) = ((@derivative R _)^[k]) :=
 begin
   induction k with k ih,
-  { rw [hasse_deriv_zero, factorial_zero, iterate_zero, one_smul, linear_map.id_coe], },
+  { rw [hasse_deriv_zero, factorial_zero, function.iterate_zero, one_smul, linear_map.id_coe], },
```

Comment:
If you put `open function` before `namespace polynomial` you'll be able to remove these `function.`. Not sure exactly why though...